notebooks/notebooks/iclr2022/checkpoint-transfer.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<p>This comment isn't really related to this PR but I feel like these experiments for subsection "Transferring to Other Checkpoints" in our paper are not well controlled enough.</p><p>The adversary trained on cp505 is actually our main passing-trick adversary that was trained on cp127-v1 at the start of its curriculum, and the adversary trained on cp127 is a different size (b10) trained on cp127-v32 for much longer.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Oh I agree that's not great. I'd guess training for longer and with larger size (b10) would help so the high-level takeaway that a network trained (solely) on cp127 does much worse against Latest will presumably transfer to other experimental setups. I think we should at the least add a caveat to this effect in the paper, and maybe we should just cut section E.3 entirely? It's already pretty buried.</p>

Reply:
Hmm I'll cut section E.3 for now. It should be fairly simple to redo these experiments and add the section back in later